qiskit/primitives/base_estimator.py

```diff
@@ -100,18 +100,28 @@
         #             <psi1(theta3)|H3|psi1(theta3)> ]
         result5 = e([0, 1, 0], [0, 1, 2], [theta1, theta2, theta3])
         print(result5)
+
+        # It is possible to pass objects.
```

Comment:
```suggestion
        # Objects can be passed instead of indices.
```

Reply:
Matching the language between the code examples for the two primitives. It can be either way, but it should be the same. https://github.com/Qiskit/qiskit-terra/pull/8065#diff-d72f1344ae63af3658830d27e2e97b7010ebade7aea9489b0d0092a37b707334R68